javascript/src/main/components/BasicCourseSearch/BasicCourseTable.js

```diff
@@ -1,8 +1,60 @@
 import React from "react";
+import { Button } from "react-bootstrap";
 import BootstrapTable from 'react-bootstrap-table-next';
 import { reformatJSON } from 'main/utils/BasicCourseTableHelpers';
 
+
+
+//const buttonFormatter = (cell, row, rowIndex, formatExtraData) => {
```

Comment:
Please remove the commented out code.

In general, commented out code should not be committed to main. Commenting out code is a strategy used during the development of code, but it should not end up in the final product.